### PR TITLE
testing/libsmi: new aport

### PR DESCRIPTION
**Pull request by mchwalisz**
https://www.ibr.cs.tu-bs.de/projects/libsmi/
Library to access SMI management information

### CODE REVIEW

File: testing/libsmi/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Contributor: Mikolaj Chwalisz <chwalisz@tkn.tu-berlin.de>
+# Maintainer: Mikolaj Chwalisz <chwalisz@tkn.tu-berlin.de>
+pkgname=libsmi
+pkgver="0.5.0"
+pkgrel=0
+pkgdesc="Library to access SMI management information"
+url="https://www.ibr.cs.tu-bs.de/projects/libsmi/"
+arch="all"
+license="Copyright (c) 1999-2008 Frank Strauss, Technical University of Braunschweig."
```
Comment (fabaff) [inline testing/libsmi/APKBUILD]: This is the copyright and not the license.  According to the COPYING file is seems to be MIT.

Comment (mchwalisz) [inline testing/libsmi/APKBUILD]: It is the same as Tcl. In Tcl package it is marked as TCL on wikipedia: BSD-style. Which one should I put in?

Comment (clandmeter) [inline testing/libsmi/APKBUILD]: Please set it to: GPLv2+ BSD

**Comment by clandmeter** [comment]
Please squash commits thx.

**Comment by clandmeter** [comment]
Not sure why this is not automatically closed, but it has already been merged.

Thanks.

**Pull request closed without merging**